master/docs/manual/configuration/schedulers.rst

```diff
@@ -641,7 +641,8 @@ The full list of parameters is:
 ``dayOfMonth`` (optional)
 
     The day of the month to start a build.
-    This defaults to ``*``, meaning every day.
+    This defaults to ``*``, meaning every day or l for last day.
```

Comment:
Let's use `L` because the matcher is not case sensitive anyway and `L` is more different from `1`, thus easier to read.

Reply:
Also needs quoting.